specs/toolsets/thrift_cppm/_build.ham

```diff
@@ -1,4 +1,4 @@
-if [ SubDirOnce TOP toolsets thrift_cppm ] = 1 { return ; }
+if [ SubDirOnce TOP specs toolsets thrift_cppm ] = 1 { return ; }
```

Comment:
Same as grpc_cppm, I think this could be an issue. My current preference would be to keep these independent - so no change, since they have different tools requirement. Ham will fail is required toolsets are missing.

Its something that could/should potentially address in the future, but its a different problem. At the very least I think it'd need its own PR.